### PR TITLE
Keep all 3rd party dependencies up-to-date

### PR DESCRIPTION
**Pull request by edigu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
         "zendframework/zendframework": "2.5.3",
         "doctrine/doctrine-orm-module": "0.9.2",
         "imagine/imagine": "0.6.3",
-        "gedmo/doctrine-extensions": "2.4.7",
-        "mathiasverraes/money": "1.3.*@dev",
+        "gedmo/doctrine-extensions": "^2.4",
+        "mathiasverraes/money": "v3.0.0-beta",
```
Comment (hkulekci) [inline composer.json]: This may not be ready for production. What do you think?

Comment (hkulekci) [inline composer.json]: Anyway, old one had been dev version, already.

Comment (edigu) [inline composer.json]: I didn't see any major BC breaks between 1.3.x dev and 3.0 versions. IMHO using boilerplate in production is responsibility of the developer.

**Comment by edigu** [comment]
We need tests.

**Comment by hkulekci** [comment]
I will

**Pull request merged**